_notices/rsn0031.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 31 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "CUDA 12 PIP Wheels Release in v23.06"

```

Comment:
The word pip is never capitalized. (See [documentation](https://pip.pypa.io/en/stable/) for examples)
```suggestion
title: "CUDA 12 pip Wheels Release in v23.06"
```